react-app/src/search-page/SearchPage.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import "./SearchPage.css"
+import FormInput from '../settings-page/FormInput';
+import AddElementButton from './AddElementButton';
+import StudentProfile from './StudentProfile';
+import notepad from './assets/notepad.svg';
+import person from './assets/person.svg';
+
+export default function SearchPage() {
+    return (
+    <div className='container'>
+        <div className = "header"></div> {/* Inset Navbar Here */}
```

Comment:
I think we'll add the navbar component through App.tsx, so we shouldn't have to add it in individual components.